trax/data/__init__.py

```diff
@@ -43,6 +44,8 @@ def data_configure(*args, **kwargs):
 Shuffle = data_configure(inputs.Shuffle)
 shuffle = inputs.shuffle
 TFDS = data_configure(tf_inputs.TFDS)
+bert_tokenizer = data_configure(bert_inputs.bert_tokenizer)
```

Comment:
We'd prefer not to export external APIs like this. Instead of this line and importing FullTokenizer below, please add a new vocab type (maybe 'bert') to the tokenize function in trax.data:
https://github.com/google/trax/blob/master/trax/data/tf_inputs.py#L303

Reply:
I added new vocab types - 'bert' and 'bert-lowercase'